robot/EDA/resources/EDA.robot

```diff
@@ -135,11 +135,11 @@ Get Affiliation Mappings
     ${NS} =                 Get EDA namespace prefix
     [Arguments]             ${field_name}
     ${result} =             SOQL Query
-    ...                     SELECT ${NS}${field_name} FROM ${NS}Affl_Mappings__c
+    ...                     SELECT ${field_name} FROM ${NS}Affl_Mappings__c
     ${count} =              Get Affl Records Count       ${field_name}
     @{ValueList}=           Create List
     FOR     ${index}        IN RANGE    ${count}
-        ${name} =               Set variable    ${result}[records][${index}][${NS}${field_name}]
```

Comment:
I'm not sure about removing the NS variable from here. It seems it was working fine previously.